loadDataIntoAstraDB/README.md

```diff
@@ -27,34 +29,32 @@ workshops
 machine_learning
 ```
 
-<<createDatabase>>
-
-It should only take a couple minutes for your database to become `ACTIVE`. Once your database becomes active, you’ll see its information appear above.
+When your database becomes active, you’ll see its information appear above. Once complete, continue in the next section.
 
 ## 2 Use the Astra data loader to upload your own dataset
-The Astra data loader is good for smaller sets of data (under 40MB), or data you want to experiment and test with. 
+The Astra data loader is good for smaller sets of data (under 40MB), or data used for experimenting and testing. 
 
 ### 2a Prepare a CSV file
-First thing, you'll need a **CSV** file to work with. We've created one for you to use as an example.
+You'll need a **CSV** file. We've created one for you to use as an example.
 
 [DOWNLOAD movies.csv](https://raw.githubusercontent.com/riptano/astra-portal-getting-started/main/loadDataIntoAstraDB/movies.csv) <-- _right click on the `DOWNLOAD` link and choose **Save Link As**_
 
-_**NOTE:** Mac users will need to explicitly set the movies file extension to .csv, otherwise it will try and download the file with *.txt._
+_**NOTE:** Mac users need to explicitly set the movies file extension to .csv; otherwise it will try and download the file with *.txt._
```

Comment:
You say "it will try" but will the system succeed with this download? If so, how about the following:

NOTE:
Mac users need to explicitly set the movies' file extenson to .csv to avoid downloading the file with *.txt.

Reply:
What you added LGTM